indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -87,7 +107,12 @@ def _process(self):
         if form.validate_on_submit():
             form.populate_obj(self.email_tpl)
             tpl = get_template_module('events/abstracts/management/_notification_tpl_list.html')
-            return jsonify_data(html=tpl.render_notification_list(self.event_new))
+            track_dict, abstract_states, contr_type_dict, email_tpl_dict = _get_rules_fields(self.event_new)
+            return jsonify_data(html=tpl.render_notification_list(self.event_new,
```

Comment:
you use the same args a few times. why not add a helper function that takes care of this?

`def _render_notification_list(event): ...`